howso/utilities/utilities.py

```diff
@@ -39,6 +39,9 @@
 # Match unescaped timezone character in datetime format strings
 SMALLEST_TIME_DELTA = 0.001
 # Regex that matches common time strings
+# <hour> AM/PM
+SIMPLE_TIME_PATTERN = r'\b([1-9]|1[0-2])\s?[aApP][mM]\b'
```

Comment:
I am not sure we should use word boundaries `\b` in either of these time patterns. You could potentially match a time in a sentence. 
i.e. as is this will currently be inferred as a time, when its not
```python
df = pd.DataFrame([
    ["10PM is the time"],
    ["Something else"],
], columns=["Desc"])
```